src/components/EditCoursePage/AdditionalMetadataFields.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import RenderInputTextField from '../RenderInputTextField';
+import FieldLabel from '../FieldLabel';
+import RichEditor from '../RichEditor';
+
+function AdditionalMetadataFields(props) {
+  const {
+    disabled,
+  } = props;
+
+  return (
+    <div className="collapsible-card pgn_collapsible mt-4">
+      <div className="collapsible-body">
```

Comment:
Make these fields required for now. Once bootcamp info is in, the required part will be determined based on the course type.